lib/info.js

```diff
@@ -108,10 +114,41 @@ module.exports = function getInfo(link, options, callback) {
     } else {
       dashmpd2 = results[1].dashmpd;
     }
-    gotConfig(options, description, config, dashmpd2, callback);
+    gotConfig(options, description, relatedVideos, config, dashmpd2, callback);
   });
 };
 
+/**
+ * @param {String} body
+ */
+function getRelatedVideos(body) {
+  var jsonStr = util.between(body, 'yt.setConfig(', '</script>');
+  var related = [];
+  if (jsonStr) {
+    try {
+      jsonStr = jsonStr.slice(0, jsonStr.indexOf(');'));
```

Comment:
If you're doing this here, why not put the string ");" in the above call to `util.between()`